packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -317,9 +327,41 @@ describe('Marketplace page', () => {
     const cloudinaryCard = screen
       .getAllByTestId('npm-package-card')
       .find((div) => div.innerHTML.includes('Cloudinary'));
-    const githubStars = queryByText(cloudinaryCard, GITHUB_STARS);
-    expect(githubStars).toBeVisible();
-    const weeklyDownloads = queryByText(cloudinaryCard, WEEKLY_DOWNLOADS);
-    expect(weeklyDownloads).toBeVisible();
+
+    const githubStarsLabel = getByLabelText(
+      cloudinaryCard,
+      `This provider was starred ${GITHUB_STARS} on GitHub`
+    );
+    expect(githubStarsLabel).toBeVisible();
+
+    const downloadsLabel = getByLabelText(
+      cloudinaryCard,
+      `This provider has ${WEEKLY_DOWNLOADS} weekly downloads`
+    );
+    expect(downloadsLabel).toBeVisible();
+  });
+
+  it('show only downloads count and not github stars if there are no or 0 stars and no downloads available for any package', () => {
+    render(App);
+    const providersTab = screen.getByRole('tab', { name: /providers/i });
+    fireEvent.click(providersTab);
+    const GITHUB_STARS = 0;
+    const WEEKLY_DOWNLOADS = 0;
+
+    const NodeMailerCard = screen
```

Comment:
You can use camelCase here instead of PascalCase since it's not a React component here but just a DOM element